packages/mdc-textfield/_mixins.scss

```diff
@@ -254,23 +271,26 @@
   @include mdc-floating-label-float-position($mdc-text-field-outlined-label-position-y);
   @include mdc-floating-label-shake-animation(text-field-outlined);
   @include mdc-text-field-outline-corner-radius_($mdc-text-field-border-radius);
+  @include mdc-states-base-color(transparent);
+  @include mdc-text-field-fill-color(transparent);
 
-  height: 56px;
+  display: inline-flex;
+  align-items: flex-end;
   border: none;
+  overflow: visible;
 
   .mdc-text-field__input {
     display: flex;
+    box-sizing: content-box;
+    height: 30px;
     padding: 12px;
     border: none;
     background-color: transparent;
     z-index: 1;
   }
 
   .mdc-floating-label {
-    @include mdc-rtl-reflexive-position(left, 16px);
-
-    position: absolute;
-    bottom: 20px;
+    width: auto;
```

Comment:
Use the `mdc-floating-label` width mixin

Reply:
Done!